doozerlib/cli/__init__.py

```diff
@@ -40,6 +40,8 @@ def print_version(ctx, param, value):
               help="Git repo or directory containing groups metadata")
 @click.option("--working-dir", metavar='PATH', default=None,
               help="Existing directory in which file operations should be performed.\n Env var: DOOZER_WORKING_DIR")
+@click.option("--registry-config", metavar='PATH', default=None,
+              help="Existing directory of DOCKER_CONFIG path, If not, when DOCKER_CONFIG env var is set, If not, the default `~/.docker/config.json` is used.\n Env var: DOCKER_CONFIG")
```

Comment:
Since `oc --registry-config` is a specific file, and `DOCKER_CONFIG` is a directory (which may contain multiple config files, even though we only care about `config.json`) and click will put both in the same variable, I think I made a mistake in suggesting that we use `--registry-config` as the parameter here. Let's just make it always a directory (which is what your code seems to expect) and call it `--registry-config-dir`.

Also I think it seems wisest to always expect `config.json` to be there, and ignore any other .json files lying around.
```suggestion
@click.option("--registry-config-dir", metavar='PATH', default=None,
              help="Directory containing docker config.json authentication; defaults to DOCKER_CONFIG env var if set, or `~/.docker/` if not.\n Env var: DOCKER_CONFIG")
```